free-3.md

```diff
@@ -9,41 +9,100 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+My idea will help people organize the tasks they need to do and create a 
+concrete plan of action that they can take. It helps people who struggle with 
+productivity but might be overwhelmed on how to overcome it. There are many
+productivity apps and task managers, but this language would allow someone
+to input their tasks and randomly schedule the tasks into a daily calendar
+based on the length, priority, and frequency of the task. Having this 
+definite scheduling puts more pressure on the user to complete the tasks and
+allow them to spend their time more efficiently.
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+A DSL would allow the user to specify the nature of the tasks they need to
+accomplish and be able to type out these tasks more efficiently. A GUI might be
+easier to navigate initially, but having the user to input tasks through code
```

Comment:
Why is it necessarily the case that inputting tasks through code allows the user to be more specific with conditions and parameters associated with each task? I can envision a GUI that functions to continue prompting the user for additional information that he/she needs to fill in. This means that the user might not have to look up documentation to ascertain what parameters he/she needs to specify.